src/mmw/apps/modeling/migrations/0035_project_layer_overrides.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2021-07-28 17:29
+from __future__ import unicode_literals
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('modeling', '0034_forbid_null_aoi_on_non_activity_projects'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='project',
+            name='layer_overrides',
+            field=django.contrib.postgres.fields.jsonb.JSONField(default={'__LAND__': 'nlcd-2011-30m-epsg5070-512-int8'}, help_text='JSON object of layers to override defaults with'),
+            preserve_default=False,
```

Comment:
This ensures that the 2011 override is used for all existing projects, but no longer.